_includes/master/reqs-sys.md

```diff
@@ -0,0 +1,44 @@
+## Node requirements
+
+- AMD64 processor
+
+- Linux kernel 3.10 or later. Kernel dependencies: `nf_conntrack_netlink` subsystem, 
+  `ip_tables` (for IPv4), `ip6_tables` (for IPv6), `ip_set`, `xt_set`, `ipt_set`, 
+  `ipt_rpfilter`, `ipt_REJECT`, `ipip` (if using {{site.prodname}} networking)
+
+  The following distributions have all of the above and are known to work well 
+  with {{site.prodname}} and {{include.orch}}.
+  - RedHat Linux 7{% if include.orch == "Kubernetes" or include.orch == "host protection" %}
+  - CentOS 7
+  - CoreOS Container Linux stable
+  - Ubuntu 16.04
+  - Debian 8
+  {% endif %}{% if include.orch == "OpenShift" %}
+  - CentOS 7
+  {% endif %}{% if include.orch == "OpenStack" %}
+  - Ubuntu 16.04
+  - CentOS 7
+  {% endif %}
+
+
+## Key/value store
+
+{{site.prodname}} {{page.version}} requires a key/value store accessible by all 
+{{site.prodname}} components. {% if include.orch == "Kubernetes" %} On Kubernetes,
+you can configure {{site.prodname}} to access an etcdv3 server directly or to
+use the Kubernetes API datastore.{% endif %}{% if include.orch == "OpenShift" %} On
+OpenShift, {{site.prodname}} can share an etcdv3 key/value store wit OpenShift, or
+you can set up an etcdv3 key/value store dedicated to {{site.prodname}}.{% endif %}
+{% if include.orch == "OpenStack" %}If you don't already have an etcdv3 server
+to connect to, we provide instructions in the [installation documentation](../installation/).{% endif %}
+
+
+## Network requirements
+
+BGP traffic over TCP must be allowed. By default, these connections use port 179.
+
+If running with IPIP encapsulation (the default mode), IPIP traffic must also be
```

Comment:
This sentence is less useful than it could be, because IPIP is connectionless and doesn't use ports (that just comes from the AWS dialog which makes you specify ports) and because it doesn't specify what proto value to use for IPIP.

I like the tabular format for this:

| Traffic | Protocol | Port (If applicable) |
|-------|------|-------|
| BGP   | TCP | 179 |
| IPIP   | 4 | N/A|

Reply:
BGP and IPIP are also protocols.....

Reply:
But I will just use "Traffic" as you suggest.